cmd/agent/app/server.go

```diff
@@ -93,7 +93,7 @@ func (a *Agent) runHealthServer(o *options.GrpcProxyAgentOptions) error {
 	muxHandler.HandleFunc("/ready", readinessHandler)
 	muxHandler.HandleFunc("/readyz", readinessHandler)
 	healthServer := &http.Server{
-		Addr:           fmt.Sprintf(":%d", o.HealthServerPort),
+		Addr:           fmt.Sprintf("%s:%d", o.HealthServerHost, o.HealthServerPort),
```

Comment:
This should use `net.JoinHostPort` to future-proof this for IPv6

Reply:
Done!